src/main/java/org/threadly/db/aurora/Driver.java

```diff
@@ -1,78 +1,41 @@
 package org.threadly.db.aurora;
 
-import java.sql.Connection;
 import java.sql.DriverManager;
-import java.sql.DriverPropertyInfo;
 import java.sql.SQLException;
-import java.sql.SQLFeatureNotSupportedException;
-import java.util.Properties;
-import java.util.logging.Logger;
-
-import org.threadly.db.AbstractArcDriver;
 
 /**
- * Threadly's AuroraArc Driver.  This Driver will create multiple connections for each returned 
- * connection it provides.  Using these connections to monitor the aurora state, and to distribute 
- * queries to multiple aurora servers when possible.
+ * Threadly's AuroraArc Driver. This Driver will create multiple connections for each returned connection it provides.
+ * Using these connections to monitor the aurora state, and to distribute queries to multiple aurora servers when
+ * possible.
  * <p>
- * This is different from most SQL drivers, in that there is shared state between all returned 
- * connections.  This shared state allows things like fail conditions to be communicated quickly, 
- * allowing for intelligence of how to mitigate problems, and use potential secondary / slave 
- * servers as soon as possible.
+ * This is different from most SQL drivers, in that there is shared state between all returned connections. This shared
+ * state allows things like fail conditions to be communicated quickly, allowing for intelligence of how to mitigate
+ * problems, and use potential secondary / slave servers as soon as possible.
  * <p>
- * In general the user does not need to concern themselves with this benefit.  Just be aware that 
- * multiple connections will be established for every connection returned by this Driver.
+ * In general the user does not need to concern themselves with this benefit. Just be aware that multiple connections
+ * will be established for every connection returned by this Driver.
  * <p>
  * Possible URL configuration options:
  * <ul>
  * <li>{@code "optimizedStateUpdates=true"} - Experimental internal code that can provide performance gains
  * </ul>
  */
-public class Driver extends AbstractArcDriver {
+public class Driver extends NonRegisteringDriver implements java.sql.Driver {
```

Comment:
Why define the `implements java.sql.Driver` here?  It is implied by the abstract driver.

Reply:
Just matter of taste.